### ISSUE TITLE
Body looks weird when underwater and flying in the air.

### ISSUE THREAD

**Issue by MidnightOG** [VictorienXP/gmod-enhanced-camera-2#23]
As i said it looks weird when flying or underwater swimming.

**Comment by Meelody520** [comment]
Yes, when I had that gravity gun or physics  gun, I could see the eyes of my character model, and it seemed to have a camera stuffed into my brain? And I've seen this problem in enhanced camera（not 2）
